ec2utils/ec2uploadimg/ec2uploadimg

```diff
@@ -549,3 +559,6 @@ except EC2UploadImgException as e:
 except Exception as e:
     print(format(e), file=sys.stderr)
     sys.exit(1)
+finally:
+    if setup:
```

Comment:
Should call clean_up unconditionally as there is no condition around the creation of the class and we do not know if any temporary resources were created. Other option would be

if not args.accountName:
  setup = ....

earlier in the code, but that would imply that we have to carry if tests in other places. Thus I prefer unconditional creation of the EC@Setup() instance and then unconditional call of clean_up().

Reply:
Removing the conditional should be fine, right! This is a left over from a previous iteration as we created the EC2Setup object later. Now we create it before the the try / except block so it should always be there.

Reply:
The conditional is actually necessary as the setup is created inside the regions for loop which means if there is no region the setup is not defined and would cause a null pointer exception.